docs/contributing.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
@christian-bromann you're saying that I should add the same front matter to this page? I had to duplicate the file in order for it to appear in the doc tree (because it seems all doc files have to be in `/docs`.